pkg/eks/api/api.go

```diff
@@ -27,10 +27,12 @@ const (
 )
 
 // SupportedRegions are the regions where EKS is available
-var SupportedRegions = []string{
-	EKSRegionUSWest2,
-	EKSRegionUSEast1,
-	EKSRegionEUWest1,
+func SupportedRegions() []string {
```

Comment:
As you cannot declare `const []string`, we prefer a function here. The previous version was written before we've adopted this convention. Just FYI :)

Reply:
Noted! 🔥